core/pkg/ingress/controller/controller.go

```diff
@@ -1065,6 +1065,28 @@ func (ic *GenericController) createServers(data []interface{},
 			servers[host].Alias = ""
 		}
 	}
+
+	// configure server snippet
+	for _, ingIf := range data {
```

Comment:
This loop is already being called in line 982, why not add this logic in there to avoid looping again?